blog/_posts/2016-01-02-new-year-resolutions.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog
+post-type: blog
+by: Martin Odersky
+title: New Year Resolutions
+---
+
+For most of us, the change of the year is an occasion for thinking
+about what we missed doing last year and where we want to improve. I decided
+there are a couple of things where I would like to do better in 2016
+than in 2015. The first is that I would like to do more blogging and
+writing in general. I have been pretty silent for most of the last
+year. This was mostly caused by the fact that I had been heads down to
+work on DOT, Scala's foundations, and _dotty_, the new Scala compiler
+platform we am working on. It's been a lot of work, but we are finally
+getting good results. DOT has now a mechanized proof of type soundness
+and the dotty compiler [can now compile
+itself](http://www.scala-lang.org/blog/2015/10/23/dotty-compiler-bootstraps.html)
+as well as large parts of Scala's standard library.
+
+The dotty compiler has a completely new and quite unusual
+architecture, which makes it resemble a functional database or a
+functional reactive program. My [talk at the JVM language
+summit](https://www.youtube.com/watch?v=WxyyJyB_Ssc) gives an
+overview. In the next months I want to write together with my
+collaborators a series of blog posts
+ that explain details of the code base in detail. The
+aim of these posts will be to present the new architectural patterns
+to a larger audience and also to help existing and potential
+contributors get familiar with the code base.
+
+My second resolution is to take a larger effort to promote simplicity
+in Scala. I believe the recent [blog post by Jim
+Plush](http://jimplush.com/talk/) should be a wakeup call for our
```

Comment:
I think the URL you mean is probably http://jimplush.com/talk/2015/12/19/moving-a-team-from-scala-to-golang/